environments/ropsten/readme.md

```diff
@@ -11,11 +11,53 @@
   - `open.aragonpm.eth`:
   `0x8980e53c979ca98e3adb8e891aebbc0f3455de37`
 - DAOFactory (Aragon 0.7): `0x3f2aa9dd22e97070518ba7988fe9b8724129d497`
+- DAOFactory (Aragon 0.8): `0x233c587095d066bafe44b543a719c93ff16423f3`
 
 ## Deployments
 
 -----------
 
+- Version: Aragon client 0.8
+- Start date: September 3rd 2019, 18.30pm CEST
+- Finish date: TBD
+- Deployer: ([@facuspagnuolo](https://github.com/facuspagnuolo)) [`0x0090aed150056316e37fe6dfa10dc63e79d173b6`](https://ropsten.etherscan.io/address/0x0090aed150056316e37fe6dfa10dc63e79d173b6)
+
```

Comment:
Can we add the aOS commit hash here as we usually do for these?

https://github.com/aragon/deployments/blob/a9fc9ba0068f249aea6d686f224762d29edf5ccf/environments/staging/readme.md#aragonos-1